tests/src/JIT/Directed/intrinsic/pow/pow1.cs

```diff
@@ -14,16 +14,18 @@ public static int Main()
         double x, y, z;
         bool pass = true;
 
-        //Check if the test is being executed on ARM
-        bool isProcessorArm = false;
+        //Check if the test is being executed on ARMARCH
+        bool isProcessorArmArch = false;
 
         string processorArchEnvVar = null;
 
         processorArchEnvVar = Environment.GetEnvironmentVariable("PROCESSOR_ARCHITECTURE");
 
-        if ((processorArchEnvVar != null) && processorArchEnvVar.Equals("ARM", StringComparison.CurrentCultureIgnoreCase))
+        if ((processorArchEnvVar != null)
```

Comment:
SGTM. Would you mind opening an issue reminding us to audit the tests for similar code?